pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -1002,7 +1002,7 @@ private String getActualColumnName(String rawTableName, String columnName,
         columnName = splits[1];
       }
       if (columnNameMap != null) {
-        return columnNameMap.getOrDefault(columnName, columnName);
+        return columnNameMap.getOrDefault(columnName.toLowerCase(), columnName);
```

Comment:
Good catch, but how the the test pass before?

Reply:
We skipped validating the response content in `OfflineClusterIntegrationTest`.